docs/core/MODULE.md

```diff
@@ -21,3 +21,118 @@ To help developers to create their own Otter compatible module, the Otter framew
 ## List of Otter official modules
 
 The full list of official modules can be found in the [Documentation Home page](../README.md#available-packages-and-tools).
+
+## How to create its own Otter module
+
+### Module development
+
+As indicated in the [definition session](#definition), the basic needs of an Otter module is to expose an `ng-add` schematics to be able to add the module via the command `ng add <package-name>`.
+> **Note**: Information relative to the `ng-add` schematics is available [here]().
+
+As any Otter module is following the same requirement, the new module can request the installation of any other by using providing helpers that will trigger the `ng-add` schematics of the requested module.
+
+Basic example of dependency module installation:
+
+```typescript
+import { chain, noop, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { NgAddSchematicsSchema } from './schema';
+
+/**
+ * Add Otter MyModule to an Otter Project
+ *
+ * @param options
+ */
+export function ngAdd(options: NgAddSchematicsSchema): Rule {
+  return async (_tree: Tree, context: SchematicContext) => {
+    try {
+      // use dynamic import to properly exception if not an Otter project
```

Comment:
Use dynamic import to properly raise an exception if it is not an Otter project.